Source/WebCore/rendering/RenderListMarker.cpp

```diff
@@ -47,7 +47,7 @@ namespace WebCore {
 WTF_MAKE_TZONE_OR_ISO_ALLOCATED_IMPL(RenderListMarker);
 
 RenderListMarker::RenderListMarker(RenderListItem& listItem, RenderStyle&& style)
-    : RenderBox(Type::ListMarker, listItem.document(), WTFMove(style))
+    : RenderBlockFlow(Type::ListMarker, listItem.document(), WTFMove(style))
```

Comment:
Inside markers should be inline boxes, not sure if you can make do with a block container.
Might be better to split `RenderListMarker` into 2 classes like in Blink?
If you want to take a look, https://blogs.igalia.com/obrufau/2020/12/21/css-marker-pseudo-element-in-chromium.html might provide some helpful info.